### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by XuluWarrior** [Wizcorp/cordova-plugin-wizviewmanager#85]
I ask because I can see that the most recent code merged into master is almost two years old.  I can see that there's a "develop" branch but that hasn't been touched for over a year now and there's no indication that it will be merged into master.

What is the state of develop?  Is there anything in it that makes it unsuitable for merging?

From my point of view I came across this plugin yesterday only to find that the example project worked but when I added similar code to my project it didn't.  It turned out that the plugin doesn't support the version of Cordova I'm using (6.3.1) on iOS.  Although looking at the Cordova it looks like it will have been broken since Cordova 4.  (See the notes on initWithWebView at https://github.com/apache/cordova-ios/blob/master/guides/API%20changes%20in%204.0.md)

I have a fix of sorts but I was hoping that this project could be brought back to life to bring it back to full compatibility with the latest versions of Cordova, iOS, Android, etc.

**Comment by jrouault** [comment]
Hello,

This project is no longer actively developed as we didn't have a chance to use it recently but contributions are always more than welcome.

Bringing this project back to full compatibility with the latest versions of Cordova/iOS/Android would indeed be the main priority. If you have any interest in this project and would be willing to make the necessary changes as a PR, we would gladly review and merge it!

As the current state of develop, there is nothing specific that makes it unsuitable for merging afaik.

If this project was to be brought back to life, it would be nice to drop the develop branch system altogether to switch to a master+releases system and publish it on NPM.

**Comment by XuluWarrior** [comment]
Sorry, I meant to reply to this and say that although I'm happy to assist with getting the project up to date, I have no experience of Objective C nor Cordova plugins architecture.  Debugging this issue was the first time I had spent any time looking at Objective C code.

I do have a fix which is good enough for my basic experiments with the plugin but I haven't started using it in anger yet and I have already noted that it breaks the existing example code which uses an older version of Cordova.

My naive fix is here
https://github.com/XuluWarrior/phonegap-plugin-wizViewManager/commit/d1e1b5a1e82f71eda33811ad7ee6401dd0cb6052

**Comment by jrouault** [comment]
Hello! Thanks for taking the time to reply. Your fix seems to be what is expected from the notes you linked to, so there should not be any issue merging it.
I tried to create a PR to discuss it but there are other commits in your `master` branch. See for yourself here:
https://github.com/Wizcorp/phonegap-plugin-wizViewManager/compare/develop...XuluWarrior:master?expand=1
If you could cherry-pick your commit based on Wizcorp's `develop` branch and PR it, I could look into fixing the example.

As a heads-up, my only comment concerning your fix would be asking why you used `self.webViewEngine.engineWebView` instead of `self.webView`?

**Comment by XuluWarrior** [comment]
Sorry, I got pulled on to other things and neglected to get back to this.

There was no reasoning behind using self.webViewEngine.engineWebView.  I was merely using code inspection to find a suitable property.  (As I said I have no Objective C or Cordova plugin experience.)  I made another check-in replacing it with self.webView.

I cherry-picked a branch but should have read your message more closely as I based it on master rather than develop.  I will recreate the pull request.

Also due to moment of madness I called my branch Issue#84 when it actually refers to Issue #86
